### PR TITLE
CSS to Less Cart Page

### PR DESCRIPTION
**Pull request by elasticSzyszko**
for review, not ready for merge

### CODE REVIEW

File: src/containers/CartPage.less [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.cart-container{
+  min-height: 100%;
+  margin-bottom: 0px;
+  :after {
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: Let's separate rules with one empty line.

Comment (dusanradovanovic) [inline src/containers/CartPage.less]: You are missing an `&`. You need to write `&:after` in order to define `:after` pseudoclass for `.cart-container`. That being said, if it looks as expected without it you might as well remove it as it is not being applied right now.

File: src/containers/CartPage.less [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+    content: "";
+    display: block;
+  }
+  
+  @media screen and (max-width: 768px) {
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: You want import `common.less` and use constants from that file as `@media (min-width: @mobileWidth) {`. Take a look at `addressform.main.less`.

Also you want to add `- 1px` for all `max-width` media queries. So final line would be:
`@media screen and (max-width: @mobileWidth - 1px) {`

File: src/containers/CartPage.less [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+      display: block;
+      width: 100%;
+    }
+    table tbody tr {
+      display: block;
+      width: 100%;
+    }
+    table tbody tr td {
+      display: block!important;
+      width: 100%!important;
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: Same as previous, avoid `!important`, if you use it put a space before it, if it looks ok the way it's written right now, try to remove the whole line. If you can remove this and previous line, then remove the whole `td` part.

File: src/containers/CartPage.less [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+
+.cart-container{
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: Space before `{`

File: src/containers/CartPage.less [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+    }
+  }
+
+  table > tbody > tr:nth-child(odd) > td,
+  table > tbody > tr:nth-child(odd) > th {
+    background-color: #e9e9e9;
+  }
+
+  .cart-container-inner{
+    :before{
```
Comment (dusanradovanovic) [inline src/containers/CartPage.less]: `&:before`

File: src/style/reset.less [hunk 6]
```diff
@@ -47,9 +47,14 @@ footer, header, hgroup, menu, nav, section {
   display: block;
 }
 
-body {
+body { //this will get overridden by bootstrap?
```
Comment (dusanradovanovic) [inline src/style/reset.less]: The order styles get included is (take a look at index.js):
- reset.less
- bootstrap
- style.less

So anything you want to keep defined on a global level goes to `style.less`.  `reset.less` merely resets the default browser styles.

**Comment by elasticSzyszko** [comment]
@dusanradovanovic ready for another review, thanks!

**Pull request closed without merging**